packages/appstore/src/components/modals/account-type-modal/account-type-modal.tsx

```diff
@@ -21,6 +21,14 @@ const financial_account: TAccountType = {
     description: localize('Trade CFDs on MT5 with forex, stock indices, commodities, and cryptocurrencies.'),
 };
 
+const swapfree_account: TAccountType = {
```

Comment:
let's move all these account types to a config/helper file

Reply:
ok